test/components/renku_data_services/connected_services/test_encryption.py

```diff
@@ -0,0 +1,93 @@
+"""Tests for the encryption in connected services."""
+
+import json
+from uuid import uuid4
+
+import pytest
+from sqlalchemy import select
+
+from renku_data_services.app_config import Config
+from renku_data_services.base_models import APIUser
+from renku_data_services.connected_services import apispec
+from renku_data_services.connected_services import orm as schemas
+from renku_data_services.utils.cryptography import decrypt_string
+
+
+@pytest.fixture
+def admin_api_user() -> APIUser:
```

Comment:
we already have an admin_user fixture in conftest that can be reused

Reply:
Done in 2a8f481361bd64a479b047af00c676f3f35d9aff.